src/main/java/org/jboss/hal/testsuite/page/config/ConfigurationPage.java

```diff
@@ -21,7 +23,7 @@
     public ConfigurationPage select(String label) {
         String cellClass = PropUtils.get("table.cell.class");
         String cellSelectedClass = PropUtils.get("table.cell.selected.class");
-        By selector = ByJQuery.selector("." + cellClass + ":contains('" + label + "')");
+        By selector = By.ByXPath.xpath("//td[contains(@class,'" + cellClass + "') and descendant::div[@class='navigation-column-item' and text()='" + label + "']]");
```

Comment:
Per discussion with @honza-kasik, we need text of subelement and there wasn't found another simple way using jquery to do equals for subelement. Thus using XPath for handling this case.